src/unstructured_client/_hooks/custom/split_pdf_hook.py

```diff
@@ -239,42 +437,10 @@ def after_error(
             self._clear_operation(operation_id)
             return (response, error)
 
-        updated_response = request_utils.create_response(responses[0], elements)
+        # updated_response = create_response(responses[-1], elements)
         self._clear_operation(operation_id)
-        return (updated_response, None)
-
-    def _await_elements(self, operation_id: str, response: requests.Response) -> Optional[list]:
-        """
-        Waits for the partition requests to complete and returns the flattened
-        elements.
-
-        Args:
-            operation_id (str): The ID of the operation.
-            response (requests.Response): The response object.
-
-        Returns:
-            Optional[list]: The flattened elements if the partition requests are
-            completed, otherwise None.
-        """
-        prt_requests = self.partition_requests.get(operation_id)
-
-        if prt_requests is None:
-            return None
-
-        responses = []
-        elements = []
-        for future in prt_requests:
-            res = future.result()
-            if res.status_code == 200:
-                responses.append(res)
-                elements.append(res.json())
-
-        if response.status_code == 200:
-            elements.append(response.json())
-
-        self.partition_responses[operation_id] = responses
-        flattened_elements = [element for sublist in elements for element in sublist]
-        return flattened_elements
+        # return (updated_response, None)
+        return (response, None)
```

Comment:
Why return the base response on error? If we want to follow the current flow it should check for at least one 200 and return error if no 200 otherwise update the response with returned element.

But I suppose it's still WIP so my comment might be meaningless :d

Reply:
outdated, resolving